pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -250,7 +250,7 @@ protected void validateAdminAndClientPermission() {
     protected void validateCreateTopic(TopicName topicName) {
         if (isTransactionInternalName(topicName)) {
             log.warn("Try to create a topic in the system topic format! {}", topicName);
```

Comment:
change the log.warn information to `log.warn("Forbidden to create transaction internal topic: {}" topicName)`  maybe more clear.